vispy/glsl/misc/spatial-filters.frag

```diff
@@ -34,29 +44,60 @@ filter2D_radius1(sampler2D texture, sampler2D kernel, float index, vec2 uv, vec2
     return filter1D_radius1(kernel, index, f.y, t0, t1);
 }
 
+float lerp(sampler2D kernel, vec2 uv) {
+    vec2 texel = uv;
+    float u = texel.x;
+    float v = texel.y;
+    float uf = fract(u);
+
+    float d0 = unpack(texture2D(kernel, vec2(u - uf, v)));
+    float d1 = unpack(texture2D(kernel, vec2(u - uf + 1, v)));
+    return mix(d0, d1, uf);
```

Comment:
I expected there to be some factor of two problem in this function, but I don't see one...

I'll try playing with the code to see if I can figure it out.